ssl/ssl_sess.c

```diff
@@ -26,54 +26,41 @@ static int remove_session_lock(SSL_CTX *ctx, SSL_SESSION *c, int lck);
 
 DEFINE_STACK_OF(SSL_SESSION)
 
-__owur static int sess_timedout(time_t t, SSL_SESSION *ss)
+__owur static ossl_inline OSSL_TIME time_t_to_ossl_time(time_t t)
```

Comment:
I think this should be in the internal/time.h header.

Reply:
I wasn't sure what to call it...

Reply:
ossl_time_time_t_to_time() matches the time to timeval function.
ossl_time_timet_to_time() would also work.